Release Notes/2019/2019-11-12-cloud-platform-release-notes.md

```diff
@@ -27,6 +27,10 @@ Application Lifecycle Management has added a search field to the size and image
 
 Additionally, the size description has been extended with the corresponding CPU and memory values to facilitate the selection of the appropriate one.
 
+##### Auto-register Terraform VM resources in managed providers
+
+Application Lifecycle Management now includes the feature to auto-register VM instances that were deployed as part of a Terraform template deployment in a vCloud-based managed provider. Compute instances included into the Terraform template box resources will be automatically registered in Cloud Application Manager and the "make managed" process will be executed on them. This is currently supported for CenturyLink Private Cloud on vCloud Foundation and VMWare vCloud Director provider types.
+
 ##### Notice of Completion: Migration for CenturyLink Cloud Shared Load Balancer Customers to Cloud Load Balancing-as-a-Service
```

Comment:
This title is wrong, please update!

Reply:
Disregard that. I see what you mean now.